docs/source/api/v4/deliveryservices_xmlid_xmlid_sslkeys_renew.rst

```diff
@@ -0,0 +1,55 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-deliveryservices-xmlid-xmlid-sslkeys-renew:
```

Comment:
this is a duplicate label. The unversioned links are always used in the latest version, so that you don't need to hunt them down throughout the rest of the docs every time we make a new API version.